context/app/static/js/components/Detail/provenance/ProvGraph/ProvGraph.jsx

```diff
@@ -1,11 +1,30 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
 
 import SectionItem from 'js/components/Detail/SectionItem';
 import { LightBlueLink } from 'js/shared-styles/Links';
+import { AppContext } from 'js/components/Providers';
+import { getImmediateDescendantProv } from 'js/hooks/useImmediateDescendantProv';
+import useProvenanceStore from 'js/stores/useProvenanceStore';
 import ProvVis from '../ProvVis';
 import { StyledTypography, FlexPaper } from './style';
 import '@hms-dbmi-bgm/react-workflow-viz/dist/react-workflow-viz.min.css';
+import ProvData from '../ProvVis/ProvData';
```

Comment:
Sorry, I'm fuzzy on the overall directory structure, but up and back down worries me a little... maybe there's a utility that should be factored to a higher directory level?